af/fastpath/fastpath/core.py

```diff
@@ -964,8 +964,13 @@ def score_web_connectivity(msm, matches) -> dict:
     #    bp = tk["body_proportion"]
     #    delta = abs((tk["body_proportion"] or 1.0) - 1.0)
     #    scores["blocking_general"] += delta
-
     # TODO: refactor to apply to all test types
+
+    if "accessible" in tk and tk["accessible"] is None:
+        # https://github.com/ooni/backend/issues/610
+        scores["accuracy"] = 0.0
+        return scores
+
```

Comment:
On re-reading, I think it would be cool to move the check for "blocking" up here as well!